cluster-up/cluster/kind/common.sh

```diff
@@ -15,6 +15,8 @@ REGISTRY_NAME=${CLUSTER_NAME}-registry
 MASTER_NODES_PATTERN="control-plane"
 WORKER_NODES_PATTERN="worker"
 
+RUN_ETCD_IN_MEMORY=${RUN_ETCD_IN_MEMORY:-"false"}
```

Comment:
All the variables that can be set outside of providers are prefixed with KUBEVIRT, let's rename this KUBEVIRT_WITH_KIND_ETCD_IN_MEMORY, also set the default to true at the end is what we want, if there is a problem we can go back to false changing just the prow job.

Reply:
Done